File: 1ºAno/Lista2/Python/Sala/files/ex6.py
a = float(input("Entre com os valores dos coeficientes da equação quadrática. Eu direi a solução se a equação houver solução:\nA = "))
b = float(input("B = "))
c = float(input("C = "))
if b ** 2 - 4 * a * c >= 0:
    print ("X' = %.2f" % (- b + ((b ** - 4 * a * c) ** 0.5) / (2 * a)))
    print ('X" = %.2f' % (- b - ((b ** - 4 * a * c) ** 0.5) / (2 * a)))
else:
    print ("A equação não possui solução :/")

File: 1ºAno/Lista2/C/Sala/files/ex6.c
#include <stdio.h>
#include <math.h>

int main() {

    double a, b, c;
    printf ("Entre com os valores dos coeficientes, vossa senhoria:\n");
    printf ("A = ");
    scanf ("%lf", &a);
    printf ("B = ");
    scanf ("%lf", &b);
    printf ("C = ");
    scanf ("%lf", &c);
    if (pow(b, 2) - 4 * a * c >= 0) {
        printf ("X' = %.2lf\n", (- b + pow(pow(b, 2) - 4 * a * c, 0.5)) / (2 * a));
        printf ("X'' = %.2lf\n", (- b - pow(pow(b, 2) - 4 * a * c, 0.5)) / (2 * a));
    }
    else {
        printf ("A equação não possui solução :(\n");
    }

    return 0;
}